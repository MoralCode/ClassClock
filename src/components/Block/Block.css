.infoBlock {
    margin: 10px 0;
}

.infoBlock * {
    margin: 1px 0;
    padding: 0;
}
