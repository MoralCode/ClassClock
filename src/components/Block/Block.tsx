import React, { Component, CSSProperties } from "react";
import "./Block.css";

interface ITextProps {
    className?: string;
    style?: CSSProperties;
}
export default class Block extends Component<ITextProps, {}> {
    render() {
        return (
            <div
                className={
                    "blockInfo" + (this.props.className ? " " + this.props.className : "")
                }
                style={this.props.style}
            >
                {this.props.children}
            </div>
        );
    }
}
