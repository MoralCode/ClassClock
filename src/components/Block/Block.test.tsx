import React from "react";
import renderer from "react-test-renderer";
import Block from "./Block";


describe("Block Component", () => {
    it("renders correctly", () => {
        const component = renderer.create(<Block />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
