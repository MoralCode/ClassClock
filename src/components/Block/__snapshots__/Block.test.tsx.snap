// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Block Component renders correctly 1`] = `
<div
  className="infoBlock"
/>
`;
